rootfs/etc/nginx/lua/configuration.lua

```diff
@@ -7,6 +7,21 @@ function _M.get_backends_data()
   return configuration_data:get("backends")
 end
 
+local function fetch_request_body()
+  ngx.req.read_body()
+  local backends = ngx.req.get_body_data()
+
+  if not backends then
+    -- request body might've been wrote to tmp file if body > client_body_buffer_size
+    local file_name = ngx.req.get_body_file()
+    local file = assert(io.open(file_name, "rb"))
+    backends = file:read("*all")
+    file:close()
```

Comment:
what happens any of this function calls fail?

Reply:
lines 18, 19 should be safe. 

`file:read` & `file:close` will return a nil if the read or close operations are problematic. 

And then since `backends` is nil, this will be handled appropriately here: https://github.com/Shopify/ingress/pull/44/files#diff-27d52aa3bacfc9feff2d5eb208af0a17R52